pages/common/rustfmt.md

```diff
@@ -0,0 +1,15 @@
+# rustfmt
+
+> Tool for formatting Rust source code.
+
+- Format a file, overwriting the original file in-place:
+
+`rustfmt {{source.rs}}`
+
+- Check a file for formating and display any changes on console:
```

Comment:
Typo: `formating` -> `formatting`. Also, perhaps `on console` would read better as `on the console`?

Reply:
Ahh ... I agree